auro-menu not working

### Discussed in https://github.com/orgs/AlaskaAirlines/discussions/561

<div type='discussions-op-text'>

<sup>Originally posted by **SDFlightOpsSupport** June 26, 2024</sup>
Hello,

I am new to Auro, and I am trying to integrate auro-select in my React App (I am using React 18) with TS and I am getting the following error :
 
Module not found: Error: Can't resolve 'mark.js/dist/mark.min' in 'D:\Github\jetstreamui\JetstreamUI\node_modules\@aurodesignsystem\auro-menu\src'
Did you mean 'mark.min.js'?

BREAKING CHANGE: The request 'mark.js/dist/mark.min' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
ERROR in ./node_modules/@aurodesignsystem/auro-menu/src/auro-menu.js 9:0-31
Module not found: Error: Can't resolve 'mark.js/dist/mark.min' in 'D:\Github\jetstreamui\JetstreamUI\node_modules\@aurodesignsystem\auro-menu\src'
Did you mean 'mark.min.js'?
BREAKING CHANGE: The request 'mark.js/dist/mark.min' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 
ERROR in ./node_modules/@aurodesignsystem/auro-menu/src/auro-menuoption.js 7:0-73
Module not found: Error: Can't resolve '@alaskaairux/icons/dist/icons/interface/check-sm_es6' in 'D:\Github\jetstreamui\JetstreamUI\node_modules\@aurodesignsystem\auro-menu\src'
Did you mean 'check-sm_es6.js'?
BREAKING CHANGE: The request '@alaskaairux/icons/dist/icons/interface/check-sm_es6' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 
webpack compiled with 2 errors
No issues found.


For the sake of Troubleshooting following are the questions that were asked to me and answers for it

Can you tell me what version of Auro-Select you have installed? 
2.10.4
![image](https://github.com/AlaskaAirlines/WC-Generator/assets/108447437/79b5dd79-37ab-4634-be18-48b94505c916)

I checked in npm website as well that is the latest version

You stated that you are using React 18 with TS, can you tell me if you are attempting to use a SSR build? 
I am using pure React which performs CSR, I am not at all using any framework like Next JS or anything else.


Can you verify that the auro-menu component has also been installed in your app? 
Yes I have. I have installed the way it was there in the [website](https://auro.alaskaair.com/components/auro/select/install).
$ npm i @aurodesignsystem/auro-select

$ npm i @aurodesignsystem/auro-dropdown
$ npm i @aurodesignsystem/auro-menu

JFYI , maybe this will be helpful, the auro-dropdown is also having issue it seems that's the reason auro-select not working??</div>

--- a/src/auro-menu.js
+++ b/src/auro-menu.js
@@ -5,8 +5,8 @@
 
 import { LitElement, html } from "lit";
 import styleCss from "./style-base-css.js";
-import './auro-menuoption';
-import "mark.js/dist/mark.min";
+import './auro-menuoption.js';
+import "mark.js/dist/mark.es6.min.js";
 
 // See https://git.io/JJ6SJ for "How to document your components using JSDoc"
 /**

--- a/src/auro-menuoption.js
+++ b/src/auro-menuoption.js
@@ -4,7 +4,7 @@
 // ---------------------------------------------------------------------
 import { LitElement, html } from "lit";
 import styleCss from "./style-menuoption-css.js";
-import check from '@alaskaairux/icons/dist/icons/interface/check-sm_es6';
+import check from '@alaskaairux/icons/dist/icons/interface/check-sm_es6.js';
 
 /**
  * The auro-menu element provides users a way to define a menu option.

